source/change-streams.rst

```diff
@@ -41,7 +41,35 @@ Terms
 Resumable Error
 ^^^^^^^^^^^^^^^
 
-Any error encountered which is not a server error, with the exception of server responses with the message “not master” or error code 43 (cursor not found).  An example might be a timeout error, or network error.
+Resumable errors consist of:
+
+- any error encountered which is not a server error (e.g. a timeout error or
+  network error)
+
+- server responses classified as "not master" or "node is recovering" errors
+  discussed in the SDAM spec's section on `Error Handling`_
```

Comment:
So as a driver writer I have to gather the set of resumable codes from by reading several specs and union-ing the various subsets together?

I guess that's OK... it prevents some duplication in the specs themselves.

Reply:
I'd be open to duplicating the error code list and string matching rules in favor of ease of reading if you feel strongly about it. Really I just want to make sure it points to the SDAM spec for the pseudo code and other rationale. I think duplicating all of that is too much.

Reply:
I understand.

My concern is that every driver writer has to collection information from more than one spec to come up with the correct set of error codes that determine retryability, resumability and SDAMness.

The opens up the door to mistakes.

In that sense it would be good if the work of collating the information from multiple specs was done only once.

Reply:
I agree, I updated to include the complete rules of error checking in each spec, and only link to SDAM as an optional reference for more information.